.github/workflows/native-image-configure-examples.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     timeout-minutes: 15
     strategy:
       matrix:
-        java-version: ['21', 'dev']
+        java-version: ['22', 'dev']
```

Comment:
I'd keep the 21 as a minimum because it's an LTS. Maybe replace `dev` with `latest-ea`?

Reply:
Let's try with `latest-ea`.